docs/application/dotnet/guides/nlp/nlp.md

```diff
@@ -0,0 +1,223 @@
+# Natural Language Processing (Nlp)
+
+
+Nlp is a subset of natural language toolkit that specifies an interface and a protocol for basic natural language processing. Tizen enables you to use Natural Language Process (Nlp) functionalities, such as language detection, parts of speech, word toeknize, and entity detection. For more information, see the [nltk Forum](http://www.nltk.org/).
+
+The main features of the Tizen.Nlp namespace include:
+
+-   Word Tokenize suppport
+
+    You can get tokens from a sentence, the type of return is WordTokenizeResult.
+
+-   Part of Speech support
+
+    You can get tokens and tags from a sentence, the type of return is PosTagResult.
+
+-   Named Entity Detection support
+
+    You can get tokens and tags from a sentence, the type of return is NamedEntityRecognitionResult.
+
+-   Language Detect support
+
+    You can get language from a sentence, the type of return is LanguageDetectedResult.
+
+## Word Tokenize
+This method break up the sentence into words and punctuation.
+
+
+## Part of Speech
+This method breaks up the sentence into words and punctuation with tags attributes.
+
+
+## Named Entity Detection
+This method breaks up the sentence into words and punctuation with tags attributes.
+
+
+## Language Detect
+This method is used to detect the language of a sentence.
+
+
+## Prerequisites
+
+Enable your application to use the Nlp functionality:
+
+1.  Using the [Tizen.Nlp](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Nlp.html) namespace, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+    ```
+    <privileges>
+       <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+       <privilege>http://tizen.org/privilege/datasharing</privilege>
+    </privileges>
+    ```
+
+2.  Using the methods and properties of the `Tizen.Nlp` namespace, include it in your application:
+
+    ```
+    using Tizen.Nlp;
+    ```
+
+## Receive Tokens and Tags - WordTokenize
+
+receive the tokens from sentence:
```

Comment:
Receive

@divyk, why are you breaking grammar?
Please start setences with capital letters.

Reply:
@divyk So, are you going to suggest fixed sentence?